railties/lib/rails/commands/server/server_command.rb

```diff
@@ -132,7 +170,17 @@ class ServerCommand < Base # :nodoc:
       def initialize(args = [], local_options = {}, config = {})
         @original_options = local_options
         super
-        @server = self.args.shift
+        @server = if using
```

Comment:
Personally I'd probably extract the message into a method like below, but it's probably fine like this :)

```ruby
@server = deprecate_positional_using(using) || options[:using]
```

Reply:
👍 I think I can extract it in a separate method.